tests/verify.test.js

```diff
@@ -77,7 +77,7 @@ describe('GET /verifyEmail', () => {
     chai
       .request(server)
       .get(`/api/users/verifyEmail/${verifytoken}`)
-      .end((err, res) => {
+      .end(async (err, res) => {
         res.should.have.status(status.BAD_REQUEST);
         res.body.errors.link.should.be.eq(errorMessages.emailLinkInvalid);
         done();
```

Comment:
why did you edit touch this file?

Reply:
These changes were implemented on @murediane PR so I had to implement it.